### DISCUSSION TITLE
Read incorrect

### DISCUSSION THREAD

**Discussion by mproper23** [jomjol/AI-on-the-edge-device#2874] [Q&A]
Hi,
I have Version 15.5.0 installed. I set up as the manual said.
![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/25171131/dd612dd2-38d6-48d3-9369-cca1fc98e690)
Why does the device Read 43.1968?
![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/25171131/7c3c9423-f8a9-4cf9-a615-3d70f49aec51)
This is the Recognition site.
Thanks for help

**Comment by spanzetta** [comment]
> Hi,
> I have Version 15.5.0 installed. I set up as the manual said.
> ![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/25171131/dd612dd2-38d6-48d3-9369-cca1fc98e690)
> Why does the device Read 43.1968?
> ![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/25171131/7c3c9423-f8a9-4cf9-a615-3d70f49aec51)
> This is the Recognition site.
> Thanks for help

I have exactly the same problem ..
Raw value is almost always correct..
Read value (after post processing) is almost always 1m3 bigger..
Using 15.6.0

**Reply by FrankCGN01** [reply]
Same Problem here. RAW value is correct, Read value is 1m³ higher than the Raw value. 
Using 15.5.0 and now 15.6.0. Problem exist in both versions. 

![Screenshot_20240210_075517](https://github.com/jomjol/AI-on-the-edge-device/assets/107265678/6cac9814-fbc6-4c39-a4a9-ac22d4aa060b)
![Screenshot_20240210_075618](https://github.com/jomjol/AI-on-the-edge-device/assets/107265678/c5220940-d026-4c0b-b904-c7b86cc09273)

**Reply by Slider0007** [reply]
> Same Problem here. RAW value is correct, Read value is 1m³ higher than the Raw value. 
> Using 15.5.0 and now 15.6.0. Problem exist in both versions. 
> 
> ![Screenshot_20240210_075517](https://github.com/jomjol/AI-on-the-edge-device/assets/107265678/6cac9814-fbc6-4c39-a4a9-ac22d4aa060b)
> ![Screenshot_20240210_075618](https://github.com/jomjol/AI-on-the-edge-device/assets/107265678/c5220940-d026-4c0b-b904-c7b86cc09273)
> 

I assume this issue here is also related to change introduced with #2778. Some further discussions, see #2857

@rainman110: FYI

**Reply by mproper23** [reply]
SO how can i fix this ? Doesn't matter if i activate or deactivate Analog/Digital Transition Start
